File: JavaNgArch/src/javangarch/domain/hibernatemapping/MyInterceptor.java
package javangarch.domain.hibernatemapping;
import org.hibernate.EmptyInterceptor;


    public class MyInterceptor  extends  EmptyInterceptor
    {

		/**
		 * 
		 */
		private static final long serialVersionUID = 1L;

		@Override
		public String onPrepareStatement(String sql) {
			return super.onPrepareStatement(sql);
		}


    } 

File: NetNgArch/src/NetNgArch.Domain/NHibernateMappingByCode/MyInterceptor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using NHibernate;

namespace NetNgArch.Domain.NHibernateMappingByCode
{
    public class MyInterceptor : EmptyInterceptor
    {

        public override NHibernate.SqlCommand.SqlString OnPrepareStatement(NHibernate.SqlCommand.SqlString sql)
        {
            return base.OnPrepareStatement(sql);

        }

    } 
}
